src/widgetastic_patternfly/__init__.py

```diff
@@ -227,39 +230,58 @@ def dismiss(self):
         for message in self.messages:
             message.dismiss()
 
-    def assert_no_error(self):
-        self.logger.info('asserting there are no error messages')
+    def match_messages(self, text=None, t=None, partial=False, inverse=False):
+        msg_type = t if isinstance(t, (tuple, list, set)) or t is None else (t, )
```

Comment:
You can simplify this check a bit by using `type(None)` so that it reads:

`msg_type = t if isinstance(t, (tuple, list, set, type(None))) else (t, )`